tests/bytes.rs

```diff
@@ -82,7 +82,10 @@ fn input() {
 
     assert_encoding_correct(&[
         Input::coin(
-            rng.gen(),
+            UtxoId {
```

Comment:
`UtxoId` should implement [Distribution](https://docs.rs/rand/0.8.4/rand/distributions/trait.Distribution.html) when `random` feature is active. This way you can keep the `rng.gen()`

Reply:
This is nice way, i did implemented it but it is not hidden behind `random` feature, i tried to correct it but it took more time then needed and just left it, it seems `Witness` has same thing.